util/util.go

```diff
@@ -43,6 +43,10 @@ func SetFdSoftLimit(newLimit uint64) error {
 	if err != nil {
 		return fmt.Errorf("SetFdSoftLimit() err: %w", err)
 	}
+	if newLimit <= rLimit.Cur {
+		// Current limit is sufficient; no need to change it.
+		return nil
+	}
```

Comment:
this looks like a special behavior, I guess need a wrapper like `SetFdSoftLimitIfLess`, or have a conditional in a place of calling?

Reply:
`RaiseRlimit` sounds good